cunumeric/eager.py

```diff
@@ -741,6 +741,11 @@ def unary_reduction(self, op, rhs, where, axes, keepdims, args, initial):
         else:
             raise RuntimeError("unsupported unary reduction op " + str(op))
 
+    def isclose(self, rhs1, rhs2, rtol, atol, equal_nan):
+        self.array[:] = np.isclose(
```

Comment:
Don't we need to do the usual boilerplate here?
```
self.check_eager_args(rhs1, rhs2)
if self.deferred is not None:
  self.deferred.isclose(rhs1, rhs2, rtol, atol, equal_nan)
  return
```

Reply:
689701c81481d30870cda552178adda1cdaea170